examples/snippets/parser/unitTestDecorator.ts

```diff
@@ -0,0 +1,34 @@
+import { orderSchema } from 'examples/snippets/parser/schema';
+import { Context } from 'aws-lambda';
+import { handler } from 'examples/snippets/parser/decorator';
+import { z } from 'zod';
+
+describe('Test handler', () => {
+  type Order = z.infer<typeof orderSchema>;
```

Comment:
I think in an actual codebase I'd import this `Order` type from the code I'm testing (similar to what you do with the schema), in this case the `safeParseDecorator.ts` file, which should export it.

This way I know that I'm being type safe and keeping them in sync